oneview_redfish_toolkit/tests/blueprints/test_session.py

```diff
@@ -0,0 +1,198 @@
+
+        self.app = self.app.test_client()
+
+        # propagate the exceptions to the test client
+        self.app.testing = True
+
+    @mock.patch.object(session_file, 'OneViewClient')
+    def test_post_session(self, oneview_client_mockup):
+        """Tests post Session"""
+
+        # Loading Session mockup result
+        with open(
+            'oneview_redfish_toolkit/mockups_redfish/Session.json'
+        ) as f:
+            session_mockup = f.read()
+
+        # Create mock response
+        oneview_client = oneview_client_mockup()
+        oneview_client.connection.get_session_id.return_value = "sessionId"
+
+        # POST Session
+        response = self.app.post("/redfish/v1/SessionService/Sessions",
+                                 data=json.dumps(dict(
+                                     UserName="administrator",
+                                     Password="password")),
+                                 content_type='application/json')
+
+        # Gets json from response
+        json_str = response.data.decode("utf-8")
+
+        # Tests response
+        self.assertEqual(status.HTTP_200_OK, response.status_code)
+        self.assertEqual("application/json", response.mimetype)
+        self.assertEqual(session_mockup, json_str)
+        self.assertIn("/redfish/v1/SessionService/Sessions/1",
+                      response.headers["Location"])
+        self.assertEqual("sessionId", response.headers["X-Auth-Token"])
+
+    @mock.patch.object(session_file, 'OneViewClient')
+    def test_post_session_invalid_key(self, oneview_client_mockup):
+        """Tests post session with an invalid JSON key"""
+
+        # Create mock response
+        oneview_client = oneview_client_mockup()
+        oneview_client.connection.get_session_id.return_value = "sessionId"
+
+        # POST Session
+        response = self.app.post("/redfish/v1/SessionService/Sessions",
+                                 data=json.dumps(dict(
+                                     InvalidKey="administrator",
+                                     Password="password")),
+                                 content_type='application/json')
+
+        # Gets json from response
+        json_str = response.data.decode("utf-8")
+
+        with open(
+                'oneview_redfish_toolkit/mockups_errors/'
+                'InvalidJsonKey.json'
+        ) as f:
+            invalid_json_key = f.read()
+
+        self.assertEqual(
+            status.HTTP_400_BAD_REQUEST,
+            response.status_code
+        )
+        self.assertEqual("application/json", response.mimetype)
+        self.assertEqual(json_str, invalid_json_key)
+
+    @mock.patch.object(session_file, 'OneViewClient')
+    def test_post_session_oneview_exception(self, oneview_client_mockup):
+        """Tests post session with HPOneViewException"""
+
+        oneview_client = oneview_client_mockup()
+
+        e = HPOneViewException({
+            'errorCode': 'HTTP_400_BAD_REQUEST',
```

Comment:
this dont make sense to me as said before

Reply:
Changed to `HTTP_401_UNAUTHORIZED`.